.github/workflows/main.yml

```diff
@@ -59,5 +65,5 @@ jobs:
         run: ./cpdb.sh upload
       
       - name: 07 Archive to secrets.EDM_DATA
-        if: github.ref == 'refs/heads/main'
+        if: github.event.inputs.archive == 'yes'
```

Comment:
I don't think it's necessary because it's set to trigger a run based on every commit.